src/azure/cli/_azure_env.py

```diff
@@ -5,9 +5,7 @@
 
 CLIENT_ID = '04b07795-8ddb-461a-bbee-02f9e1bf7b46'
 
-ENV_DEFAULT = 'AzureCloud'
-ENV_US_GOVERNMENT = 'AzureUSGovernment'
-ENV_CHINA = 'AzureChinaCloud'
+ENV_DEFAULT = 'dogfood'
```

Comment:
Curious, why didn't you just add one more env instead of replacing the existing ones..